test/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity.UnitTests.Tests/ProjectModifierTests.cs

```diff
@@ -0,0 +1,206 @@
+                    (
+                        SyntaxFactory.InvocationExpression
+                        (
+                            SyntaxFactory.MemberAccessExpression
+                            (
+                                SyntaxKind.SimpleMemberAccessExpression,
+                                SyntaxFactory.IdentifierName
+                                (
+                                    SyntaxFactory.Identifier
+                                    (
+                                        SyntaxFactory.TriviaList
+                                        (
+                                            SyntaxFactory.Whitespace("    ")
+                                        ),
+                                        "app",
+                                        SyntaxFactory.TriviaList()
+                                    )
+                                ),
+                                SyntaxFactory.IdentifierName("UseRouting")
+                            )
+                            .WithOperatorToken
+                            (
+                                SyntaxFactory.Token(SyntaxKind.DotToken)
+                            )
+                        )
+                        .WithArgumentList
+                        (
+                            SyntaxFactory.ArgumentList()
+                            .WithOpenParenToken
+                            (
+                                SyntaxFactory.Token(SyntaxKind.OpenParenToken)
+                            )
+                            .WithCloseParenToken
+                            (
+                                SyntaxFactory.Token(SyntaxKind.CloseParenToken)
+                            )
+                        )
+                    )
+                    .WithSemicolonToken
+                    (
+                        SyntaxFactory.Token
+                        (
+                            SyntaxFactory.TriviaList(),
+                            SyntaxKind.SemicolonToken,
+                            SyntaxFactory.TriviaList
+                            (
+                                SyntaxFactory.LineFeed
+                            )
+                        )
+                    )
+                )
+            )
+            .WithOpenBraceToken
+            (
+                SyntaxFactory.Token
+                (
+                    SyntaxFactory.TriviaList(),
+                    SyntaxKind.OpenBraceToken,
+                    SyntaxFactory.TriviaList
+                    (
+                        SyntaxFactory.LineFeed
+                    )
+                )
+            )
+            .WithCloseBraceToken
+            (
+                SyntaxFactory.Token(SyntaxKind.CloseBraceToken)
+            );
+
+            StatementSyntax statement = SyntaxFactory.ParseStatement("app.UseRouting();");
+            Assert.True(projectModifier.StatementExists(blockSyntax, statement));
```

Comment:
>blockSyntax [](http://example.com/codeflow?start=56&length=11)

instead of doing it with code - do you want to use parse too? like you did for "statemet" , would be easier i guess..

Reply:
fixed.